<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>131</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="LI_Username"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LI_Password"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LI_Password_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="L_Username">
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="L_Password">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="L_Password_2">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="P_Signup">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认注册</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
